### ISSUE TITLE
wasm: Image stretches when the UWP behaviour is to maintain ratio

### ISSUE THREAD

**Issue by yowl** [nventive/Uno#469]
## I'm submitting a...

- Bug report (I searched for similar issues and did not find one)
 

## Current behavior
When using the `<Image>` tag in wasm and resizing the browser, the image is also resized.  In UWP the image maintains its size and aspect ratio.

## Expected behavior
Image should not change as browser is resized as per UWP behaviour

## Minimal reproduction of the problem with instructions
Clone https://github.com/yowl/UnoStandardRefTest/tree/wasmimagestretch and run the UWP and wasm heads.  Resize both and not the visual changes.

## Environment

```
Nuget Package: 
Uno.UI
Package Version(s): 1.43.0-dev.570

Affected platform(s):
- [ ] iOS
- [ ] Android
- [ x] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [x] 2017 (version: 15.9.4)
- [ ] 2017 Preview (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )
```

**Comment by jeromelaban** [comment]
This may have been fixed by #472. Could you revalidate with latest master ?

**Comment by yowl** [comment]
Thanks, indeed it is no longer stretched out of orginal aspect ration.  Thanks!
